examples/vue/package.json

```diff
@@ -21,6 +21,6 @@
     "typescript": "^4.3.2",
     "vite": "^2.4.2",
     "vite-plugin-voie": "^0.7.3",
-    "vue-tsc": "^0.0.24"
+    "vue-tsc": "^0.3.0"
```

Comment:
Just curious, what is this package used for?

Reply:
It's used to check vue's typescript files